packages/common/src/utils/route.ts

```diff
@@ -155,7 +155,7 @@ export const FAVORITING_USERS_ROUTE = '/favoriting_users'
 export const FOLLOWING_USERS_ROUTE = '/following'
 export const FOLLOWERS_USERS_ROUTE = '/followers'
 export const LEADERBOARD_USERS_ROUTE = '/leaderboard'
-export const COIN_DETAIL_ROUTE = '/coins/:ticker/details'
+export const COIN_DETAIL_ROUTE = '/coins/:ticker'
```

Comment:
is this even being used anywhere?

Reply:
ohh but it's dup'd

Reply:
sry @Kyle-Shanks

Reply:
realized we actually do need it for mobile web bc the on-chain details is in a separate pge